### ISSUE TITLE
OSError: libcusparse.so.11: cannot open shared object file: No such file or directory  CUDA Setup failed despite CUDA being available.

### ISSUE THREAD

**Issue by Aashi779** [TimDettmers/bitsandbytes#1234]
### System Info

Windows, 3.10.14, Installed RAM - 8.00 GB (7.75 GB usable), System type - 64-bit operating system, x64-based processor 

### Reproduction

python CRAG_Inference.py --generator_path model/hf-models/model-selfrag/selfrag_llama2_7b --input_file knowledge_prep/test_rbi.txt --output_file output_file --internal_knowledge_path knowledge_prep/ref/correct --external_knowledge_path knowledge_prep/ref/incorrect --combined_knowledge_path knowledge_prep/ref/ambiguous --task eli5 --method crag --device cuda:0 --ndocs 4 --batch_size 1 --upper_threshold 0.592 --lower_threshold 0.995 - I was running this file.

### Expected behavior

It should have loaded the generator(selfrag-llama2 7b in this case), but it threw the above error

**Comment by TimDettmers** [comment]
Closing this issue. The error (`libcusparse.so.11: cannot open shared object file`) on a Windows path indicates a CUDA version mismatch on an old bitsandbytes build. Note that `.so` files are Linux shared libraries — this error on Windows means the build was not Windows-compatible. Official Windows support was added in v0.43.0, and modern releases ship platform-specific wheels that handle CUDA version detection reliably.

If you're still hitting this on the **latest** bitsandbytes (v0.45+), please open a new issue with the output of `python -m bitsandbytes` and your environment details.
